WcaOnRails/app/webpacker/components/StaticPages/TeamsCommittees.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Button, Icon, Popup } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import UserBadge, { subtextForMember, subtextForOfficer } from '../UserBadge';
+import '../../stylesheets/static_pages/teams_committees.scss';
+
+function TeamsCommittees({ officers = [], teams = [], officerTitles = [] }) {
+  return (
+    <>
+      <h1>{I18n.t('about.structure.teams_committees_councils')}</h1>
+      <p>
+        {I18n.t('about.structure.committees')}
+      </p>
+
+      <h3 id="officers">{I18n.t('about.structure.officers.name')}</h3>
```

Comment:
What's this id used for?

Reply:
Nothing as far as I can tell! Must have slipped in from Rails, my bad.